### ISSUE TITLE
Homebrew is issuing a warning for the clj-kondo tap

### ISSUE THREAD

**Issue by pmonks** [clj-kondo/clj-kondo#2860]
**version**

N/A

**platform**

N/A

**problem**

Homebrew is displaying this message:
```text
Warning: Skipping clj-kondo: tap formula is not trusted
```

**repro**

```shell
brew cleanup
```

**expected behavior**

Homebrew does not emit a warning.

**Comment by borkdude** [comment]
Fixed
